downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -47,105 +42,101 @@ A few backward-incompatible changes have been introduced during the Swan Lake Be
 
 Annotations are now supported on tuple members. Tuple members can be annotated using annotations declared with the `field` attachment point.
 
-```ballerina
-annotation annot on field;
+  ```ballerina
+  annotation annot on field;
 
-type T [int, @annot string];
-```
+  type T [int, @annot
+      string ];
+  ```
```

Comment:
@MaryamZi Is this code formatting correct?

Reply:
No, can be on the  same line here. I believe this was changed when there were bullet points previously.

@praneesha, can you fix the indentation for the entire code block and also the line after. Doesn't have to be indented?

Reply:
This was what I got after doing `Format document` on the VS Code editor. Will check and fix.

Reply:
It's probably because you were using an older version of Ballerina instead of the RC.

Reply:
Fixed.